lib/xchain/provider/fetch.go

```diff
@@ -106,6 +107,7 @@ func (p *Provider) GetBlock(ctx context.Context, chainID uint64, height uint64)
 
 	// ignore if our height is greater than the finalized height
 	if height > finalisedHeader.Number.Uint64() {
+		log.Warn(ctx, "Trying to fetch block height > finalized height", nil, "chain", chainID, "attested height", height, "finalized height", finalisedHeader.Number.Uint64())
```

Comment:
this provider doesn't know anything about attestations so we should avoid that in the log, just remove "attested"